tests/integration/test_assessment.py

```diff
@@ -179,13 +182,24 @@ def single_choice_question(self, number, mentoring, controls, choice_name, resul
 
         self.selected_controls(controls, last)
 
-        controls.submit.click()
```

Comment:
This is not a fix, rather a test refactoring. There was a big utility method that: 

1. Did some setup 
2. Clicked on submit 
3. Done some assertions

I spitted that into two methods.